src/server/identity/cmds/cmds_test.go

```diff
@@ -79,8 +75,8 @@ func TestGetSetConfig(t *testing.T) {
 	tu.ActivateAuth(t)
 	defer tu.DeleteAll(t)
 	require.NoError(t, tu.BashCmd(`
-		pachctl idp set-config --issuer 'http://example.com:1234'
-		pachctl idp get-config | match 'issuer: "http://example.com:1234"' 
+		echo '{"issuer": "http://example.com:1234"}' | pachctl idp set-config 
```

Comment:
does `deserializeYAML()` also parse JSON? I'm confused by what code path is deserializing this json.

Reply:
JSON is a strict subset of YAML, so if you support YAML you also suppot JSON. I used JSON in the tests because the whitespace is hard to format correctly.

Reply:
wow I can't believe I didn't know that! that's cool